### ISSUE TITLE
Retrieve author names by language

### ISSUE THREAD

**Issue by ingoboerner** [dracor-org/dracor-api#150]
related to #149 and (maybe) #127:
Is there a function to retrieve author names by language? I tested it with `afinogenov-mashenka`: english version of the name is put into the also name as array; which is already helpful, but there is no way of classifying the alternative names by type or language

```
map {
    "refs": [map {
        "ref": "Q729569",
        "type": "wikidata"
    }],
    "shortname": "Афиногенов",
    "name": "Афиногенов, Александр Николаевич",
    "alsoKnownAs": ["Alexander Afinogenov"],
    "fullname": "Александр Николаевич Афиногенов",
    "key": "wikidata:Q729569"
}
```
it could be solved by adding alternative name forms as maps as well, like:

`"alsoKnownAs": [{"name": "Alexander Afinogenov", "lang": "eng" ,... }]`

or: maybe have another function, which would it make backwards compatible

**Comment by cmil** [comment]
In #118 we discussed adding `nameEn` and `fullnameEn`. Would that be sufficient?

**Comment by ingoboerner** [comment]
yes, sure, sounds like a good solution to me! and I can assume, that the first `<persName>` is always in the language of the play. Which means, when I request the map by the dutil function `"shortname"`, `"name"` and `"fullname"` are  in the language of the play as well, right? 

In the RDF – see #43 – I would then add the language tag to the `rdfs:label` when I create it for the author; so each author node would have two rdfs:labels, one in the language of the play, which would include the value of the field `"name"`. The second `rdfs:label` would be labeled as english and have `"nameEn"` as it's value.

**Comment by ingoboerner** [comment]
could I assume, that the map has always the fields "nameEn" ... ? Or will this be the case only if there are more `<tei:persName>` elements in `<tei:author>` (e.g. in rusdracor).

**Comment by cmil** [comment]
The `*En` fields will only be provided when there actually is an English alternative, i.e. a respective element with `xml:lang="eng"`. Currently the author name functions that accept the `$lang` parameter do not fall back to the first `persName` when the language passed is the same as the document language. That means if you want the names in the original language you should use the function without passing a `$lang` parameter. 

I think we need a place where we document assumptions like this (e.g. that we expect the first `persName` to be in document language) as soon as possible.

**Comment by cmil** [comment]
The solution implemented in #151 has been released with [v0.82.0](https://github.com/dracor-org/dracor-api/releases/tag/v0.82.0). I'm closing the issue. @ingoboerner please reopen if something is missing.
